src/applications/mhv/landing-page/selectors/index.js

```diff
@@ -5,21 +5,26 @@ import {
 import { isLoggedIn, selectProfile } from '~/platform/user/selectors';
 import { selectDrupalStaticData } from '~/platform/site-wide/drupal-static-data/selectors';
 
-import { isLandingPageEnabled } from './isLandingPageEnabled';
+import { isLandingPageEnabled, personalizationEnabled } from './featureToggles';
 import { isLandingPageEnabledForUser } from './isLandingPageEnabledForUser';
 import { hasHealthData } from './hasHealthData';
 
 const selectVamcEhrData = state =>
   selectDrupalStaticData(state)?.vamcEhrData || {};
 
+const selectGreetingName = state =>
```

Comment:
I am thinking this is really not a selector. Maybe move this somewhere else?

Reply:
It is selecting data from state.